imap_processing/swe/l1b/swe_l1b.py

```diff
@@ -41,17 +35,8 @@ def swe_l1b(l1a_dataset: xr.Dataset):
         conversion_table_path=conversion_table_path,
         packet_name=packet_name.name,
     )
-    if apid == SWEAPID.SWE_SCIENCE:
-        data = swe_l1b_science(eu_data)
-        if data is None:
-            print("No data to write to CDF")
-            return
-        # TODO: replace "sci" with proper descriptor, or add to global attributes
-        data.attrs["descriptor"] = "sci"
-    else:
-        data = eu_data
-        # Update global attributes to l1b global attributes
-        data.attrs.update(swe_cdf_attrs.swe_l1b_global_attrs.output())
-        # TODO: replace "sci" with proper descriptor, or add to global attributes
-        data.attrs["descriptor"] = "sci"
+    data = swe_l1b_science(eu_data)
+    if data is None:
+        print("No data to write to CDF")
+        return
```

Comment:
Do we want this as an info/warning instead? Also, an empty `return` still returns `None`, so is the same here as `return data`, which is `return None` in this case. Alternatively, would you want to put this logging/info print into the swe_l1b science routine itself to say we didn't get any data there?

```suggestion
        logger.info("No data to write to CDF")
```